actors/runtime/src/test_utils.rs

```diff
@@ -965,51 +965,64 @@ impl Syscalls for MockRuntime {
     ) -> anyhow::Result<Cid> {
         let exp =
             self.expectations.borrow_mut().expect_compute_unsealed_sector_cid.take().ok_or_else(
-                || actor_error!(ErrIllegalState; "Unexpected syscall to ComputeUnsealedSectorCID"),
+                || actor_error!(illegal_state; "Unexpected syscall to ComputeUnsealedSectorCID"),
```

Comment:
The mock is totally messed up here. It should not be returning ActorError for mock expectation violations.